CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ Versions
 * Use only major Node version to name node images
 * Add Node 12
 * Upgrade ScoutSuite to 5.3.3
+- Upgrade Golang version: 1.13
```

Comment:
Can we stick to one style of bullet points, using the same as the lines above?

Reply:
fixed :blush: